.github/workflows/copyright.yaml

```diff
@@ -0,0 +1,35 @@
+name: copyright
+on:
+  pull_request:
+    paths:
+      - 'src/**'
+      - '!**.md'
+  push:
+    paths:
+      - 'src/**'
+      - '!**.md'
+
+jobs:
+  main:
+    name: Enforce copyright notices
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 2.7
```

Comment:
Should we use Python 3 here? The script looks quite simple, I'm sure it wouldn't be _that_ much effort to port.

Reply:
as per comment, agree but let's do it in a py3- specific PR